module2/Homework2-start/ColorPicker/ContentView.swift

```diff
@@ -0,0 +1,167 @@
+import SwiftUI
+
+struct ContentView: View {
+  @Environment(\.verticalSizeClass) var verticalSizeClass
+  @Environment(\.horizontalSizeClass) var horizontalSizeClass
+
+  @State private var alertIsVisible: Bool = false
```

Comment:
Is this just a cut/paste issue? I don't see you using it.

Reply:
yes, good catch.